views/getting_started.dt

```diff
@@ -1,116 +1,109 @@
-		li The two configurations "application" and "library" get added automatically, depending on the availability of those files
-		li A folder <code>views</code> is automatically treated as a string import folder (the <code>-J</code> command line switch for DMD)
+	pre.code $ dub init myproject
 
+	p This begins an interactive session:
 
-	h2#installation Installation
+	pre.code.
+		Package recipe format (sdl/json) [json]:
+		Name [myproject]:
+		Description [A minimal D application.]: My first project
+		Author name [imadev]: My Name
+		License [proprietary]: Boost
+		Copyright string [Copyright © 2017, imadev]:
+		Add dependency (leave empty to skip) []:
+		Successfully created an empty project in '/home/imadev/src/myproject'.
+		Package successfully created in myproject
 
-	p Simply <a href="download">download</a> the pre-compiled package for your platform. On Windows, the installer will perform all the necessary setup. On other systems, the only thing that you may want to do, other than extracting the archive, is to place the <code>dub</code> executable somewhere in your PATH so that you can call it from anywhere.
+	p DUB has created a "myproject" directory containing a .gitignore, a dub configuration file, and a source/app.d source tree.
 
-	p If a binary is not available for your platform, simply clone or download the <a href="https://github.com/rejectedsoftware/dub">dub repository</a> and run the contained <code>./build.sh</code>. You need to have DMD and libcurl development files installed (Only DMD on Windows).
-
-
-	h2#foreign-projects Building foreign projects
-
-	p Building existing projects is as simple as getting the package and running either <code>dub run</code> (or short <code>dub</code>) to build and run or <code>dub build</code> from the package's root directory to build it. DUB will automatically download all needed dependencies and issue the build command. See the <a href="#command-line">command line documentation</a> for information on how to customize the build process.
-
-	p To get the package, either use <code>git clone</code> with the projects git URL, or type <code>dub fetch &lt;package name&gt;</code> to have the package downloaded and installed in a package repository in your user folder. <code>dub run &lt;package name&gt;</code> can then be used to build and execute the package. <code>dub fetch --local &lt;package name&gt;</code> can be used instead to extract the package into a sub folder of the current working directory.
-
-	h2#own-projects Creating an own project
-
-	p The easiest way is to type <code>dub init &lt;package name&gt;</code> on the command line to have DUB create an application skeleton for you. It will create the following directory structure:
+	p Notice that there are two configuration file formats available. JSON is a commonly-known format, and SDL (SDLang) is a clean, minimalist format. Both offer equivalent functionality (though unlike JSON, SDLang allows comments); use whichever you prefer.
```

Comment:
Instead of the semicolon (preceding "use whichever you prefer"? If not, where would you put it?